app/guid-node/metadata/index/template.hbs

```diff
@@ -0,0 +1,42 @@
+<div local-class='metadata-container {{if this.isMobile 'mobile'}}'>
+    <div local-class='header-container'>
+        <div local-class='header'>
+            <h2 data-test-metadata-header>
+                {{t 'metadata.header'}}
+            </h2>
+        </div>
+        <div local-class='actions'>
+            <div local-class='download spacer'>
+                <div local-class='spacer'>
+                    <FaIcon @icon='download' />
+                </div>
+                <div>
+                    {{t 'metadata.download'}}
+                </div>
+            </div>
+            <div local-class='add-schema'>
+                <div local-class='spacer'>
+                    <FaIcon @icon='plus' />
+                </div>
+                <div>
+                    <OsfLink
+                        data-test-add-cedar-schema
+                        data-analytics-name='guid-node.metadata.add'
```

Comment:
The `data-analytics-name` attributes should be human-readable, so maybe something like:
```suggestion
                        data-analytics-name='Add metadata schema'
```